src/ucm/mmap/install.c

```diff
@@ -185,32 +189,27 @@ ucs_status_t ucm_mmap_test_events(int events, int *out_events)
     ucm_fire_mmap_events_internal(events, &data);
     ucm_event_handler_remove(&handler);
 
-    *out_events = data.out_events;
-
-    ucm_debug("mmap test: got 0x%x out of 0x%x", *out_events, events);
+    ucm_debug("mmap test: got 0x%x out of 0x%x", data.out_events, events);
 
     /* Return success if we caught all wanted events */
-    if (!ucs_test_all_flags(*out_events, events)) {
+    if (!ucs_test_all_flags(data.out_events, events)) {
         return UCS_ERR_UNSUPPORTED;
     }
 
     return UCS_OK;
 }
 
-/* Called with lock held */
-static ucs_status_t ucm_mmap_test(int events)
+ucs_status_t ucm_mmap_test_installed_events(int events)
 {
-    static int installed_events = 0;
-    int out_events              = 0; /* GCC bug: it reports compilation fail if not initialized */
     ucs_status_t status;
 
-    if (ucs_test_all_flags(installed_events, events)) {
-        /* All requested events are already installed */
-        return UCS_OK;
-    }
-
-    status            = ucm_mmap_test_events(events, &out_events);
-    installed_events |= out_events;
+    /*
+     * return UCS_OK iff all installed events are actually working
```

Comment:
iff->if

Reply:
iff means "if and only if", do you think it's not applicable here?

Reply:
uuuups, looks like typo :)
then it is ok :)